v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -366,7 +366,14 @@ func (f *Frontend) setupChromium() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = settings.PutIsZoomControlEnabled(false)
+
+
+	if opts := f.frontendOptions.Windows; opts != nil && opts.ZoomFactor > 0.0 {
+		log.Print(fmt.Sprintf("Zoom: %d", opts.ZoomFactor))
+		chromium.PutZoomFactor(opts.ZoomFactor)
+	}
+
+	err = settings.PutIsZoomControlEnabled(true)
 	if err != nil {
 		log.Fatal(err)
 	}
```

Comment:
Should we put this section within the test for a zoom factor in case a value is not set?

Reply:
the IsZoomControlEnabled allows the user to change the zoom. The ZoomFactor initializes the webview2 with a specific scale. The default zoomFactor is to 1.0 (by our code).

IsZoomControlEnabled can be true without setting a ZoomFactor. In that case it will be 1.0 and the user can change it at wish.

It is also possible to do not allow the user to change the zoomFactor but sets one using the config (f.e. for a kiosk like app, to fit a screen nicely).

As you pointed out, this is what I still have to add to this PR, in the documentation. It is not complicated but with no documentation it can be confusing :) Thoughts?

Reply:
Should this be `true` or `options.IsZoomControlEnabled`?

Reply:
Fixed he logic and add the option as parameter to the COM API call, both ZoomFactor.